presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -417,9 +417,9 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         hiveClientConfig.isS3SelectPushdownEnabled(),
                         false),
                 booleanProperty(
-                        STREAMING_AGGREGATION_ENABLED,
-                        "Enable streaming aggregation execution",
-                        hiveClientConfig.isStreamingAggregationEnabled(),
+                        ORDER_BASED_EXECUTION_ENABLED,
+                        "Enable order-based execution. When it's enabled, hive files become non-splittable",
```

Comment:
And the ordering property of the table is exposed to the optimizer to plan streaming aggregation and merge join when possible.